flexmeasures/api/v3_0/accounts.py

```diff
@@ -0,0 +1,102 @@
+from flask_classful import FlaskView, route
+from webargs.flaskparser import use_kwargs
+from flask_security import current_user, auth_required
+from flask_json import as_json
+
+from flexmeasures.auth.policy import user_has_admin_access
+from flexmeasures.auth.decorators import permission_required_for_context
+from flexmeasures.data.models.user import Account
+from flexmeasures.data.services.accounts import get_accounts
+from flexmeasures.api.common.schemas.users import AccountIdField
+from flexmeasures.data.schemas.account import AccountSchema
+
+"""
+API endpoints to manage accounts.
+
+Both POST (to create) and DELETE are not accessible via the API, but as CLI functions.
+Editing (PATCH) is also not yet implemented, but might be next, e.g. for the name or roles.
+"""
+
+# Instantiate schemas outside of endpoint logic to minimize response time
+account_schema = AccountSchema()
+accounts_schema = AccountSchema(many=True)
+partial_account_schema = AccountSchema(partial=True)
+
+
+class AccountAPI(FlaskView):
+    route_base = "/accounts"
+    trailing_slash = False
+
+    @route("", methods=["GET"])
+    @auth_required("token", "session")
+    @as_json
+    def index(self):
```

Comment:
Idea for a follow-up ticket: allow filtering by account role (e.g. `role="<service> subscriber"`). `get_accounts` already supports passing an account role, so this feature would just require writing `data.schemas.account.AccountRoleField` using marshmallow. Can you label it with "good first issue"?

Reply:
Tip: the comment's menu (...) has an option called "Reference in new issue".